### DISCUSSION TITLE
Github errors on opening a folder

### DISCUSSION THREAD

**Discussion by pon4atina** [community/community#44807] [Repositories]
### Select Topic Area

Bug

### Body

I just created a new repo and noticed an error while trying to open a folder. After some fiddling I reduced it to this: trying to open a folder starting with `AD_`, you get an error like this (`AD_SOMETHING` works like this)
![image](https://user-images.githubusercontent.com/89482481/213526864-f82d61e8-5972-460e-9d61-ebf653849406.png)
After that I created a [blank repo](https://github.com/pon4atina/test_repo) and I got the same error. So I believe this is a bug as I couldn't find anything forbidding me from doing this

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pon4atina ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by pon4atina** [reply]
Oh, thank you!
